backend/core/src/main/java/org/sonarsource/sonarlint/core/analysis/AnalysisService.java

```diff
@@ -698,7 +697,6 @@ private void streamIssue(String configScopeId, UUID analysisId, Issue issue, Con
     if (activeRule != null) {
       var rawIssue = new RawIssue(issue, activeRule);
       rawIssues.add(rawIssue);
-      client.didRaiseIssue(new DidRaiseIssueParams(configScopeId, analysisId, toDto(issue, activeRule)));
```

Comment:
If I remember correctly this is about file level issues that we don't track because they don't have a location. So we decide to not raise them?

Reply:
I think you made this change originally. I don't think it's related to file-level issues, it's just the method we were using at some point to report raw issues one-by-one. It's not useful anymore. But maybe I forgot something